Essentials/src/com/earth2me/essentials/commands/Commandthunder.java

```diff
@@ -18,22 +18,21 @@ public Commandthunder() {
 
     @Override
     public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
-        if (args.length < 1) {
+        if (args.length == 0) {
             throw new NotEnoughArgumentsException();
         }
 
         final World world = user.getWorld();
         final boolean setThunder = args[0].equalsIgnoreCase("true");
-        if (args.length > 1) {
-
-            world.setThundering(setThunder);
-            world.setThunderDuration(Integer.parseInt(args[1]) * 20);
-            user.sendMessage(tl("thunderDuration", (setThunder ? tl("enabled") : tl("disabled")), Integer.parseInt(args[1])));
-
-        } else {
+        if (args.length == 1) {
             world.setThundering(setThunder);
```

Comment:
This method doesn't need to be called twice

Reply:
Which method?

Reply:
Ignore, was blind and didn't see the return.